DirectProgramming/C++SYCL_FPGA/Tutorials/Features/memory_attributes/src/memory_attributes.cpp

```diff
@@ -14,7 +14,7 @@ using namespace sycl;
 constexpr size_t kRows = 8;
 constexpr size_t kVec = 4;
 constexpr size_t kMaxVal = 512;
-constexpr size_t kNumTests = 64;
+constexpr size_t kNumTests = 2;
```

Comment:
Should this restriction only be applied to the simulator flow?

Reply:
This is a great idea! I was not aware that this is possible.
I wonder do you know how to do this in the code?

Reply:
```suggestion
#if defined (FPGA_SIMULATOR)
constexpr size_t kNumTests = 2;
#else
constexpr size_t kNumTests = 64;
#endif
```

Reply:
Thank you Yohann, let me commit this and do the testing again locally

Reply:
Testing looks good